tests/phpunit/tests/db.php

```diff
@@ -1511,6 +1509,21 @@ public function test_charset_switched_to_utf8() {
 		$this->assertSame( 'utf8_general_ci', $result['collate'] );
 	}
 
+
```

Comment:
```suggestion
```
Extra empty line here.

Reply:
Great catch @SergeyBiryukov. Resolved in https://github.com/WordPress/wordpress-develop/pull/3541/commits/b0443f4cc8e68ec9a3538ed45b23bcecf9842197.